internal/testutil/acc/advanced_cluster_test.go

```diff
@@ -0,0 +1,32 @@
+package acc_test
+
+import (
+	"testing"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConvertToTPFAttrsMap(t *testing.T) {
+	t.Setenv(config.AdvancedClusterV2SchemaEnvVar, "true")
```

Comment:
Why is this necessary?

Reply:
you're right that it's confusing, I've moved all tests for TPF helper funcs to the same file so they can be run in GH: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2862/commits/984e868ab19b8755d367d153fae9004a24b4e47b